openownership/data-standard
# Docs: Generating statements page replaces statement id page, fixes #615

# Overview

**What does this pull request do?**

- Replaces the old 'Statement Identifiers' page and 'Updating Statement' pages with a 'Generating Statements' page
- Fixes a build error on the examples/index.rst page
- Renames 'Representing information updates' page to just 'Information updates'
- Adds error correction and redaction guidance to the 'Information updates' page
- Fixes references and anchors in record-ids.rst and reference.rst

**How can a reviewer test or examine your changes?**

Build the docs to check for build errors or warnings. There will be one warning re the reference ':any:`guidance-dates`' in the 'Information updates' page (since the page with this anchor has yet to be added).

Review the new/updated text on the 'Generating statements' and 'Information updates' pages.

**Who is best placed to review it?**

@kathryn-ods 

Closes #615 

## Translations

- [x] N/A I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [x] N/A I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
